odoo/addons/base/models/ir_attachment.py

```diff
@@ -742,4 +742,4 @@ def regenerate_assets_bundles(self):
             ('res_id', '=', 0),
             ('create_uid', '=', SUPERUSER_ID),
         ]).unlink()
-        self.clear_caches()
+        self.env.registry.clear_caches()
```

Comment:
`clear_caches` doesn't exists on the registry, or I miss something ? Did you mean `clear_all_caches` ? If the goal is to clean the cache of asset you need to call `clear_cache('asset')` (without `s`).

Reply:
That was a typo when transfering over from old to new system. Fixed.